modules/diary.py

```diff
@@ -35,6 +35,14 @@ def now_date():
     return str(time.strftime("%d-%m-%Y"))
 
```

Comment:
please remove all files other than this one, since they are not related to the fix.

Reply:
okay, will do that

Reply:
Okay, I that case, I will close this PR